pkgs/development/python-modules/memory-allocator/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cython
+}:
+
+buildPythonPackage rec {
+  pname = "memory-allocator";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "memory_allocator";
+    sha256 = "sha256-UUcR71e3eAQIQpmWM+AVQxVtgHvrNjaIlHo5pURUln0=";
+  };
+
+  propagatedBuildInputs = [
+    cython
+  ];
```

Comment:
Missing checkInputs and pythonImportsCheck.

Reply:
See above :)

Reply:
pythonImportsCheck should be added anyway.

Reply:
OK, added. But, to be clear, the package does have a test suite comprised of 35 tests, and the default `setuptoolsCheckPhase` causes those tests to be executed.